cpp/leaky_stdlib.h

```diff
@@ -87,8 +87,9 @@ inline void _exit(int status) {
   ::_exit(status);
 }
 
-inline void write(int fd, Str* s) {
-  ::write(fd, s->data_, len(s));
+inline size_t write(int fd, Str* s) {
+  size_t size = ::write(fd, s->data_, len(s));
```

Comment:
Is this to shut up a warning?

I think what we need to do is check `result < 0` and then throw an exception

That's how Python works so we should match the semantics

This will just push the ignored return value higher up

Reply:
Alright, I'll revert this change.